### ISSUE TITLE
Support config files passed on the command line

### ISSUE THREAD

**Issue by istreeter** [snowplow-incubator/snowplow-bigquery-loader#282]
Currently the loader apps only accept base64-encoded configuration passed directly on the command line:

```
--config=<BASE64 CONFIG HERE> --resolver=<BASE64 CONFIG HERE>
```

However, all other Snowplow applications allow passing configuration from files:

```
--config=/path/to/config --resolver=/path/to/resolver
```

Some Snowplow apps even support both, so I suggest we do that.  The file-based method is more convenient in some deployment setups, e.g. using Kubernetes with ConfigMaps.

**Comment by tnightengale** [comment]
Also, the docs incorrectly present passing file paths in the examples. But it clearly does not work. The docs for Snowplow are probably the laziest I have ever seen.
<img width="1440" alt="Screen Shot 2022-06-17 at 4 55 13 PM" src="https://user-images.githubusercontent.com/17347282/174410423-87f0614d-2ff4-4a90-b348-247cd5326200.png">

**Comment by istreeter** [comment]
Hi @tnightengale, I have just [released version 1.4.0](https://github.com/snowplow-incubator/snowplow-bigquery-loader/releases/tag/1.4.0) of BigQuery loader which adds in support for config files on the command line.  It was a missing feature that had been annoying me for a while, so I'm pleased to have addressed it; I hope you find it helpful too.

I acknowledge your point about the Snowplow docs, and I understand exactly why you say this.  I promise we are slowly slowly working to improve the docs, but we have plenty more still to do.  If you are struggling with the docs you are always welcome to ask us questions over [on our Discourse site](https://discourse.snowplowanalytics.com/); all questions are welcome and we try to be as helpful as possible.
